android/src/main/java/com/swmansion/rnscreens/bottomsheet/BottomSheetBehaviorExt.kt

```diff
@@ -0,0 +1,42 @@
+package com.swmansion.rnscreens.bottomsheet
+
+import android.view.View
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+
+internal fun <T : View> BottomSheetBehavior<T>.useSingleDetent(height: Int? = null, forceExpandedState: Boolean = true): BottomSheetBehavior<T> {
+    this.skipCollapsed = true
+    this.isFitToContents = true
+    if (forceExpandedState) {
+        this.state = BottomSheetBehavior.STATE_EXPANDED
+    }
+    height?.let { maxHeight = height }
+    return this
+}
+
+internal fun <T : View> BottomSheetBehavior<T>.useTwoDetents(
+    @BottomSheetBehavior.StableState state: Int? = null,
+    firstHeight: Int? = null,
+    secondHeight: Int? = null
```

Comment:
Wouldn't it be better to stick with the original "peekHeight" and "maxHeight"?
Same applies to `useSingleDetent` and `useThreeDetents`.

Reply:
My intention was to abstract the `BottomSheetBehavior` properties away as much as possible, because `peekHeight` is not the "first detent height` and `maxHeight` is not `secondHeight` in general. They serve these functions only when other properties are set appropriately. 

I've yet failed to abstract it away from useThreeDetents, because situation is a little bit more complicated there & requires another refactor, which will come in the future.